code/game/objects/items/weapons/tools/tape.dm

```diff
@@ -126,24 +126,36 @@
 	..()
 	flags |= NOBLUDGEON
 
+/obj/item/weapon/ducttape/update_plane()
+	..()
+	update_icon()
+
+
 /obj/item/weapon/ducttape/examine(mob/user)
 	return stuck.examine(user)
 
 /obj/item/weapon/ducttape/proc/attach(var/obj/item/weapon/W)
 	stuck = W
 	W.forceMove(src)
-	if (istype(W, /obj/item/weapon/paper))
-		icon_state = W.icon_state + "_taped"
-		overlays = W.overlays
+	update_icon()
+	name = W.name + " (taped)"
+
+/obj/item/weapon/ducttape/update_icon()
```

Comment:
why is duct tape modified here?

Reply:
Duct tape allows you to tape items to the walls and uses some plane magic for this